include/netlink/netlink.h

```diff
@@ -77,16 +77,19 @@ extern int			nl_recvmsgs_default(struct nl_sock *);
 extern int			nl_wait_for_ack(struct nl_sock *);
 
 extern int			nl_pickup(struct nl_sock *,
-					  int (*parser)(struct nl_cache_ops *,
-						struct sockaddr_nl *,
-						struct nlmsghdr *,
-						struct nl_parser_param *),
+					  int (*)(struct nl_cache_ops *,
+						  struct sockaddr_nl *,
+						  struct nlmsghdr *,
+						  struct nl_parser_param *),
```

Comment:
Merged this individual patch to `main` ([[1]](https://github.com/thom311/libnl/commit/bdfdf738b57a1607df7d5143f04a0691b3d62be5)).